Cargo.toml

```diff
@@ -379,6 +379,16 @@ description = "Demonstrates pixel perfect in 2d"
 category = "2D Rendering"
 wasm = true
 
+[[example]]
+name = "pixel_grid_snap_2d"
+path = "examples/2d/pixel_grid_snap_2d.rs"
```

Comment:
I think the _2d suffix is redundant here.

Reply:
Oh, right, I initially named it `render_to_texture_2d` to differentiate from the original example, but forgot to remove it. Thank you!